### DISCUSSION TITLE
[Icon Request] Pin or tack

### DISCUSSION THREAD

**Discussion by techenby** [tailwindlabs/heroicons#454] [Ideas]
For representing a pinned item

**Comment by jzillmann** [comment]
Double that!

**Comment by santyas** [comment]
Please add a pin or tack to symbolize an attached message ie. Thanks!

**Comment by itstimwhite** [comment]
Yes please! We have a feature to "pin" items to the top of a list.  There seems to be no good icon for this at the moment so we're sourcing outside svgs.

**Comment by ordago** [comment]
hey @itstimwhite , I'm using https://tabler-icons.io to supplement heroicons

they have a pin icon

**Reply by itstimwhite** [reply]
Checking that out, thanks!

**Comment by NatoBoram** [comment]
A pin icon is useful in various circumstances

![image](https://github.com/tailwindlabs/heroicons/assets/10495562/511680ad-24dd-4c12-bfe5-30e3f73c8f07)

![image](https://github.com/tailwindlabs/heroicons/assets/10495562/3e82961e-7880-4c25-b34f-2a8a49d168ae)


# 📌

**Comment by AyubM** [comment]
Bump

I know there are a lot of people requesting icons for their niche use case which doesn't make sense, but I was surprised there's no icon for a pin included.

"Pinning" something is literally the word used, and it is a common enough task in many applications.

**Comment by sayhicoelho** [comment]
Any updates?

**Reply by alexmillman23** [reply]
I've switched: https://tabler.io/icons

**Comment by felinusfish** [comment]
**Please** add a pushpin icon. I love this icon library but I just can't stay with it if it's been four years and we still don't have an icon as simple as this. There are a few more icons I'd really want to see but THIS is a given and makes it honestly frustrating to use this library.

**Comment by ndri** [comment]
I added `pin` and `pin-slash` to [Sidekickicons](https://github.com/ndri/sidekickicons/), which are meant to be used as a companion to Heroicons. Browse them here: 
- https://sidekickicons.com/?icon=pin
- https://sidekickicons.com/?icon=pin-slash

**Reply by sanake-dev** [reply]
Thank youuuu

**Comment by sanake-dev** [comment]
2026 we still asking for a pin icon please you have the best icons library.
